contracts/DefaultStrategy.sol

```diff
@@ -630,16 +666,22 @@ contract DefaultStrategy is UUPSOwnableUpgradeable, UsingRegistryUpgradeable, Ma
         uint256 groupsIndex;
 
         for (groupsIndex = 0; groupsIndex < maxGroupCount; groupsIndex++) {
-            if (votesAmount == 0 || votedGroup == address(0)) {
+            if (votedGroup == depositGroupToIgnore) {
```

Comment:
Is this necessary? Shouldn't we already have a mechanism anyways that skips overflowed groups?

Reply:
The problem is when group fulfils following conditions:

- strategy is specific strategy
- strategy is active strategy and also is the tail of default strategy
- we are depositing to this group through specific strategy and group will overflow with this deposit

In this scenario we will schedule as much votes as possible to this group through specific strategy and then we will request default strategy to schedule rest. Since this group is also tail of default strategy it will try to schedule votes to this group since votes from specific strategy are not yet really scheduled.